config/warp_tokens.ts

```diff
@@ -11,6 +11,8 @@ export const warpRouteConfig: WarpRouteConfig = {
     type: TokenType.native, //  TokenType.native or TokenType.collateral
     // If type is collateral, a token address is required:
     // address: '0x123...'
+    // If the token is an NFT (ERC721), set to true:
+    // isNft: boolean
```

Comment:
Can we not piggy back upon the token type here?

Reply:
No, there's no differentiation of subType in TokenType. 
We could double the enum (NativeFungible, NativeNonFungible) but that seems worse IMO. This is what the warp uI does currently

Reply:
@yorhodes can correct me here, but I thought `syntheticUri` and `collateralUri` are the NFT token types

Reply:
URI is specifically for NFTs with metadata
@jmrossy is right